lib/pages/dashboard_page.dart

```diff
@@ -0,0 +1,96 @@
+import 'package:clone_instagram/constants/SourceString.dart';
+import 'package:flutter/material.dart';
+
+class DashboardPage extends StatefulWidget {
+  const DashboardPage({super.key});
+
+  @override
+  State<DashboardPage> createState() => _DashboardPageState();
+}
+
+class _DashboardPageState extends State<DashboardPage> {
+  int _selectedIndex = 0;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(
+          title: const Text(SourceString.titleAppBar),
+        ),
+        body: const Center(
+          child: Text(""),
```

Comment:
Body should be the Screen that was selected

Reply:
I changed